### ISSUE TITLE
How do we simulate "Real Time" interaction without WebSockets ?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-aws-lambda#32]
WebSockets allow us to stream additional data from/to client/server beyond the limits of http request/response. This has opened a world of Real Time apps. 
Given that keeping Lambda functions running for long time will cost a lot (_because we are billed per 100ms_) we need to consider an _alternative_ approach for sending data to the client _after_ the initial page/data is rendered... 

**_Polling**_? 

http://stackoverflow.com/questions/12555043/my-understanding-of-http-polling-long-polling-http-streaming-and-websockets

**Comment by nelsonic** [comment]
could we just use **_Pusher**_ https://pusher.com/features to do our WebSockets for us and still remain "**_Serverless**_" ...?
![pusher-features](https://cloud.githubusercontent.com/assets/194400/12650468/e2d15226-c5da-11e5-943c-8c4b725776ff.png)

**Comment by nikhilaravi** [comment]
@nelsonic interesting! 
Do you mean something like  DynamoDB update -> triggers Lambda -> publishes message on Pusher channel -> Client receives message -> repeat
So Lambda function is only run for very short periods.

**Comment by nelsonic** [comment]
Yeah... what do you think? we could _try_ it and see what people think. 
The cost won't be _that_ much. 10k Concurrent Connections is "_Only_" $499/month.
It sounds like a lot to us but to a company used to spending hundreds of thousands on Hardware & DevOps teams, its a _bargain_....
